sycl/test/devicelib/c99_complex_math_fp64_test.cpp

```diff
@@ -1,6 +1,6 @@
 // UNSUPPORTED: windows
 // RUN: %clangxx -fsycl -c %s -o %t.o
-// RUN: %clangxx -fsycl %t.o %llvm_build_libs_dir/libsycl-complex-fp64.o -o %t.out
+// RUN: %clangxx -fsycl %t.o %sycl_libs_dir/libsycl-complex-fp64.o -o %t.out
```

Comment:
Why do we need to use `%sycl_libs_dir/` prefix?
I thought that clang driver is aware about location of deployed libraries. Am I wrong? 
Do we use non-standard directories?
Why?

Reply:
I'm not an expert in device libraries and clang driver. But for me it looks like that technically `libsycl-complex-fp64.o` is not a library, but some object file. The clang driver seems ok to find deployed `libsycl.so` library, but not these object files.
Tagging @asavonic , @mdtoguchi , @AGindinson who may know a bit more than me.

Reply:
as `libsycl.so` is considered a library, the linker will go ahead and search for it via the either the default `-L<dir>` values or `-L<dir>` values passed on the command line.  Objects need to be fully qualified or in the `$CWD`

Reply:
I expect this functionality to be implemented as a library and handled by the driver.
Let's discuss it separately.